rxgroups/src/main/java/com/airbnb/rxgroups/ProcessorUtils.java

```diff
@@ -0,0 +1,59 @@
+package com.airbnb.rxgroups;
```

Comment:
I think the problem with `compileOnly` is that it will still appear on the classpath during development. So someone could accidentally reference `compileOnly` code in a non processor class, and cause a runtime crash.